### PR TITLE
Feat/card config content

### PR DESCRIPTION
**Pull request by Damien-MATHIEU**
#61 

## Description

feat: added config details and button to hide it

### CODE REVIEW

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  .headersListContainer {
+    flex-wrap: wrap;
+    flex-direction: column;
+    max-width: 100%;
+    overflow-x: auto;
+  }
+
+  tr {
+    border-bottom: 1px solid #6a6a6a;
```
Comment (nartydev) [inline src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts:21]: Maybe you can add `#6a6a6a` in colors constant file

File: src/components/CardConfig/subcomponents/CardConfigContent/index.tsx [hunk 2]
```diff
@@ -1,12 +1,84 @@
...
+          </Text>
+          <Text weight={FontWeights.MEDIUM}>Concurrency</Text>
+        </div>
+        <div className="mr-8">
+          <Text
+            font={FontFamilies.POPPINS}
+            weight={FontWeights.BOLD}
+            size={FontSizes.LARGE}
+          >
+            50ms
```
Comment (nartydev) [inline src/components/CardConfig/subcomponents/CardConfigContent/index.tsx:58]: The value must be a props

File: src/components/CardConfig/subcomponents/CardConfigContent/index.tsx [hunk 3]
```diff
@@ -1,12 +1,84 @@
...
+        Options
+      </Text>
+      <div className="f mb-3">
+        <div className="mr-7">
+          <Text
+            font={FontFamilies.POPPINS}
+            weight={FontWeights.BOLD}
+            size={FontSizes.LARGE}
+          >
+            1
```
Comment (nartydev) [inline src/components/CardConfig/subcomponents/CardConfigContent/index.tsx:48]: The value must be a props

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  keyValueCouple {
+    border-top: none;
+  }
+
+  .keyColumn {
+    width: 30%;
+    padding: ${SPACINGS[1]};
+    border-right: 1px solid #6a6a6a;
```
Comment (nartydev) [inline src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts:37]: And here

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  tr {
+    border-bottom: 1px solid #6a6a6a;
+    border-left: 1px solid #6a6a6a;
+    border-right: 1px solid #6a6a6a;
+  }
+
+  thead {
+    border-top: 1px solid #6a6a6a;
```
Comment (nartydev) [inline src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts:27]: Same here

**Review by nartydev — CHANGES_REQUESTED** [review]

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts (hunk 1 above)
Comment (Damien-MATHIEU) [reply src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts]: Done

File: src/components/CardConfig/subcomponents/CardConfigContent/index.tsx (hunk 2 above)
Comment (Damien-MATHIEU) [reply src/components/CardConfig/subcomponents/CardConfigContent/index.tsx]: Done

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts (hunk 4 above)
Comment (Damien-MATHIEU) [reply src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts]: Done

File: src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts (hunk 5 above)
Comment (Damien-MATHIEU) [reply src/components/CardConfig/subcomponents/CardConfigContent/core/cardConfigContent.styles.ts]: Done

File: src/components/CardConfig/index.tsx [hunk 6]
```diff
@@ -13,7 +13,11 @@ export const CardConfig: FC = () => {
           <CardConfigHeader />
         </ExpansionPanel.Header>
         <ExpansionPanel.Content>
-          <CardConfigContent className="mt-3" />
+          <CardConfigContent
+            className="mt-3"
+            concurrency={1}
+            interval={'50ms'}
```
Comment (nartydev) [inline src/components/CardConfig/index.tsx:19]: For string you don't need to wrap with bracket you can write :
`interval='50ms'`

Comment (Damien-MATHIEU) [reply src/components/CardConfig/index.tsx:19]: Done

**Review by nartydev — APPROVED** [review]

**Pull request merged**